python/mxnet/image.py

```diff
@@ -37,8 +37,8 @@ def imdecode(buf, **kwargs):
         1 for three channel color output. 0 for grayscale output.
     to_rgb : int, optional, default=1
         1 for RGB formatted output (MXNet default). 0 for BGR formatted output (OpenCV default).
-    out : NDArray, optional
-        Output buffer. Use `None` for automatic allocation.
+
+    # removed out parameter as it is not used
 
     Returns
     -------
```

Comment:
I have fixed this at https://github.com/dmlc/mxnet/pull/6134 

Will update the doc once that is pushed